src/box/lua/schema.lua

```diff
@@ -2537,9 +2537,9 @@ base_index_mt.select_ffi = function(index, key, opts)
     check_select_safety(index, key_is_nil, iterator, limit, offset, fullscan)
     local region_svp = builtin.box_region_used()
     normalize_position(index, after, iterator_pos, iterator_pos_end)
-    nok = builtin.box_select(index.space_id, index.id, iterator, offset, limit,
-                             key, key_end, iterator_pos, iterator_pos_end,
-                             fetch_pos, port) ~= 0
+    nok = builtin.box_select_ffi(index.space_id, index.id, key, key_end,
```

Comment:
Is there a ticket for this? #7739? Should we re-enable LuaJIT fro the tests disabled in the scope of this issue?

https://github.com/tarantool/tarantool/blob/e02bc87feba73ce42e00719f1d40f2c0d363dc72/test/engine-luatest/pagination_test.lua#L10-L19

Should we file a ticket to revert this change once the LuaJIT issue is fixed and reference it in the code?

Anyway, I think that this fix should be reviewed by someone from the Lango team.

Reply:
No, #7739 is different. This workaround is for #7946

Reply:
This PR will close #7946

Reply:
> Anyway, I think that this fix should be reviewed by someone from the Lango team.

Requested a review from @igormunkin

Reply:
> Should we re-enable LuaJIT fro the tests disabled in the scope of this issue?

JIT engine is unrelated for this case. But I wonder, whether we can run all M1 tests with JIT disabled... I'll check if there is nothing left related to JIT for M1 in the queue. All in all, one can just add `-e jit.off() jit.flush()` to the command launching the Tarantool.

> Should we file a ticket to revert this change once the LuaJIT issue is fixed and reference it in the code?

Up to you. Then there will be `N+1` M1-related tickets on GitHub. I have no idea when they are scheduled to RnD plans. Furthermore, I'd rather believe Mike Pall regarding the potential fix for this problem (without M1 ABI spec that is more verbose than [this one](https://developer.apple.com/documentation/xcode/writing-arm64-code-for-apple-platforms)).